src/Network/Network/Az.Network.psd1

```diff
@@ -104,6 +104,13 @@ CmdletsToExport = 'Add-AzApplicationGatewayAuthenticationCertificate',
                'Set-AzApplicationGatewayWebApplicationFirewallConfiguration', 
                'New-AzApplicationGatewayFirewallDisabledRuleGroupConfig', 
                'New-AzApplicationGatewayFirewallExclusionConfig', 
+			   'New-AzApplicationGatewayFirewallCondition',
+			   'New-AzApplicationGatewayFirewallCustomRule',
+			   'New-AzApplicationGatewayFirewallMatchVariable',
+			   'New-AzApplicationGatewayFirewallPolicy',
+			   'Get-AzApplicationGatewayFirewallPolicy',
```

Comment:
Where were these cmdlets in the design review? Not seeing them here: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/253, and looks like they do not fit into our piping guidelines: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/piping-in-powershell.md. I will reopen the cmdlet design review, please add a new comment with the unlisted cmdlets.